### DISCUSSION TITLE
Add office plugin

### DISCUSSION THREAD

**Discussion by lcings** [DustinBrett/daedalOS#342] [Feature Requests]
Can you add OnlyOffice or other similar plugins? There are many requirements for office documents.

**Comment by DustinBrett** [comment]
Likely not OnlyOffice as that doesn't seem to be free. But indeed I will/am considering possible office apps. The TinyMCE app is a decent Word replacement as it's a WYSYWIG editor. For Excel I am planning to eventually add something like SheetJS or something like that. Other possible options are things like LibreOffice WASM (https://wasm-test.libreoffice.org/) but it's not a small program to download and has a lot of issues. For ODF I was thinking of https://webodf.org/. Anyway, I have plans but it will take a while. Until then I have document editors and viewers of various types, but there are indeed gaps. I also plan to add third party app support eventually which could have others adding these kinds of apps. It's also possible for anyone to fork the code and add these apps, and if they were well done enough I would consider upstreaming them.

**Reply by lcings** [reply]
Great!I am looking forward to this coming.
